lib/active_remote/primary_key.rb

```diff
@@ -48,7 +48,8 @@ def primary_key
     #   person = Person.new(1)
     #   person.to_key # => [1]
     def to_key
-      send(primary_key) ? [send(primary_key)] : nil
+      has_primary_key = respond_to?(primary_key.to_sym) && send(primary_key)
```

Comment:
is `to_sym` necessary? seems like the check for `has_primary_key` is cacheable?

Reply:
It's not, thanks for pointing that out.  As far as caching it, ActiveModel doesn't when they do something similar which gave me pause, but I couldn't think of a reason why not.